build-heads.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
```

Comment:
Please add:

* `set -eu` for "strict mode"

* `cd "$(dirname "$0")"` to ensure we are in the script directory.